app-nordigen/services/nordigen-service.js

```diff
@@ -0,0 +1,363 @@
+import BankFactory from '../bank-factory.js';
+import {
+  RequisitionNotLinked,
+  AccountNotLinedToRequisition,
+  InvalidInputDataError,
+  InvalidNordigenTokenError,
+  AccessDeniedError,
+  NotFoundError,
+  ResourceSuspended,
+  RateLimitError,
+  UnknownError,
+  ServiceError
+} from '../errors.js';
+import * as nordigenNode from 'nordigen-node';
+import uuid from '@actual-app/api/app/bundle.api.js';
+const NordigenClient = nordigenNode.default;
+const nordigenClient = new NordigenClient({
+  secretId: process.env.SECRET_ID,
+  secretKey: process.env.SECRET_KEY
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could we rename these to `NORDIGEN_SECRET_*` to be more explicit about what exactly these are?

Reply:
Or include them in the `config.json` file and use env vars prefixed with `ACTUAL_` like the rest of those ones?

Reply:
That's an even better suggestion. +1 to what Jed said